### ISSUE TITLE
Smart Content (provider "pages") should return the nodeType of the nodes

### ISSUE THREAD

**Issue by thomasduenser** [sulu/SuluHeadlessBundle#96]
When using the `smart_content` type, the response doesn't contain the `nodeType` of the child pages. This would be helpful to open those pages e.g. in a new tab.

Tested with following example:
```
<property name="smartContent" type="smart_content">
            <meta>
                <title lang="de">Smart Content</title>
                <title lang="en">Smart Content</title>
            </meta>
            <params>
                <param name="provider" value="pages"/>
                <param name="categories_parameter" value="c"/>
                <param name="website_categories_operator" value="AND"/>
                <param name="properties" type="collection">
                    <param name="excerptTitle" value="excerpt.title"/>
                    <param name="excerptCategories" value="excerpt.categories"/>
                    <param name="excerptImages" value="excerpt.images"/>
                    <param name="headerTitle" value="headerTitle"/>
                    <param name="headerImage" value="headerImage"/>
                </param>
            </params>
        </property>
```

**Comment by niklasnatter** [comment]
Thanks for creating this issue! This should be tackled together with https://github.com/sulu/SuluHeadlessBundle/issues/62

**Comment by niklasnatter** [comment]
Thanks for reporting! Should be fixed by #98
